src/test/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/EmailServiceTest.java

```diff
@@ -38,6 +39,14 @@
     private static final String AUTH_USER = EMAIL + "; forename=demoForename; surname=demoSurname";
     private static final String REASON = "THIS IS A REASON";
 
+    private static final String EMAIL_RECIPIENTS_CARDIFF_TEST =
+            "COTIndividual@companieshouse.gov.uk,COTExttest@companieshouse.gov.uk,dissolution@companieshouse.gov.uk";
```

Comment:
Would suggest using clearly fake data here as this code is on a public repo. I don't think anything here is unreasonable but probably best something like "Fake1@Cardiff.com"

Reply:
Renamed